tests/JustSaying.UnitTests/Messaging/Channels/MessageReceiveBufferTests/WhenSubscriberIsSlow.cs

```diff
@@ -39,8 +39,7 @@ public WhenSubscriberIsSlow(ITestOutputHelper testOutputHelper)
             TimeSpan.FromSeconds(1),
             queue,
             sqsMiddleware,
-            new MessageReceiveToggle(),
-            TimeSpan.FromMilliseconds(100),
+            null,
```

Comment:
As we have the constructor overload, these test files should be able to just use the existing one? That way the overall number of files touched on the final PR diff should be less.

Reply:
Ah, sorry, I must have confused it with `JustSayingBus`.

FWIW, my preference is that files _do_ have the final new line (so you don't get the 🚫 symbol in GitHub diffs) so where it's been _added_ I wouldn't worry about it too much.

Reply:
No worries!
That makes it easier then 🙂